app/helpers/iso-formatter.ts

```diff
@@ -1,4 +1,4 @@
-import { format, parseISO } from 'date-fns';
+import { lightFormat, parseISO } from 'date-fns';
```

Comment:
The output of `lightFormat` is the same in our use-cases, and it's a lot smaller (3KB vs 24KB minified).